src/commands/new.js

```diff
@@ -136,6 +142,7 @@ const cmd = {
 
     if (npmOptions.npmInstall === 'Yeah, please' || npmOptions.npmInstall === 'yes') {
       await exec('npm install')    
+      await exec('npx eslint \"**/*.{js,jsx}\" --fix')
```

Comment:
if we run it without an eslint file configured, it will ask to install it.
so, if I said before (line 69) that i don't want to use eslint on my project and it(npx eslint) installs the eslint when I generate the project, we will provide a bad DX for programmer 

We need to run it only if the project use eslint (when generated or after it)